app/src/main/kotlin/vergecurrency/vergewallet/viewmodel/ShowPaperkeyViewModel.kt

```diff
@@ -5,15 +5,15 @@ import vergecurrency.vergewallet.service.model.EncryptedPreferencesManager.Compa
 import vergecurrency.vergewallet.service.model.MnemonicManager
 
 class ShowPaperkeyViewModel : ViewModel() {
-    var seed: Array<String>
+    var seed: Array<ByteArray>
```

Comment:
That's an array inside an array, why?

Reply:
mnemonic words

Reply:
ah you have an array for all the words and then we have an array inside of it for each char byte encoded. Alright.